### PR TITLE
[Common] Update Product Name in Main Directory

### PR DESCRIPTION
**Pull request by dilyantraykov**

### CODE REVIEW

File: common-ui-automation.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Telerik UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_nam
 
 ## AutomationMode property
 
-With Q2 2014 SP release of UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} you will have the option to turn off the generating of the automation peers through the new global __AutomationMode__ property of the __AutomationManager__.
+With Q2 2014 SP release of Telerik UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} you will have the option to turn off the generating of the automation peers through the new global __AutomationMode__ property of the __AutomationManager__.
```
Comment (martinivanoff) [inline common-ui-automation.md]: You can use {{ site.framework_name }} here

File: eqatec-analytics-application-integration.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ position: 14
...
+When you are creating an application for a broad audience, integrating some kind of analytics framework is crucial, because you will need to analyze the usage data of the application and its features and most probably you will need to know about any application crashes or other errors occurred during the execution. With Q3 2013 release Telerik UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} provides integrated support for [Telerik Analytics](http://www.telerik.com/analytics). Using Analytics you will be able to trace certain features of the controls and get statistics about their usage.
```
Comment (martinivanoff) [inline eqatec-analytics-application-integration.md]: You can use {{ site.framework_name }} here.

File: eqatec-analytics-application-integration.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Both options are explained in details below.
       
 ## How to Create a New Project Supporting Analytics
      
-Using the [Telerik Visual Extensions]({%slug radcontrols-for-silverlight-vs-extensions-project-configuration%}) you can quickly create an application configured to use UI for Silverlight.
+Using the [Telerik Visual Extensions]({%slug radcontrols-for-silverlight-vs-extensions-project-configuration%}) you can quickly create an application configured to use Telerik UI for Silverlight.
```
Comment (martinivanoff) [inline eqatec-analytics-application-integration.md]: UI for Silverlight? Isn't this article shown also in WPF?

File: eqatec-analytics-application-integration.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ where [product_key] will be replaced with the entered in the __New Project Wizar
     
 ## How to Add Support to Existing Applications
       
-First, you will need to reference the __EQATEC.Analytics.Monitor.dll__ and __Telerik.Windows.Analytics.dll__ in your application. Note that these dlls are distributed together with the other assemblies in the UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} suite.        
+First, you will need to reference the __EQATEC.Analytics.Monitor.dll__ and __Telerik.Windows.Analytics.dll__ in your application. Note that these dlls are distributed together with the other assemblies in the Telerik UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} suite.
```
Comment (martinivanoff) [inline eqatec-analytics-application-integration.md]: You can use {{ site.framework_name }} here.

File: eqatec-analytics-application-integration.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ Clicking on the __Session__ link will show additional details about the session
 If you go to the __Feature Use link__ in the Dashboard panel, you will find information and statistics for the registered features for different periods of time:
 ![eqatec dashboard 4](images/eqatec_dashboard_4.png)
 
-## Traceable Features in UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %}
+## Traceable Features in Telerik UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %}
```
Comment (martinivanoff) [inline eqatec-analytics-application-integration.md]: You can use {{ site.framework_name }} here.

File: introduction.md [hunk 6]
```diff
@@ -41,32 +41,32 @@ License agreement and detailed information about different licenses and which on
 
 ## Installation and Deployment
 
-To learn how to install UI for {% if site.site_name == 'Silverlight' %}Silverlight{% if site.site_name == 'WPF' %}WPF{% endif %}{% endif %} and for comprehensive list of our controls and their assemblies you can visit the [Installation and Deployment]({%slug installation-installing-which-file-do-i-need%}) section.
+To learn how to install Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% if site.site_name == 'WPF' %}WPF{% endif %}{% endif %} and for comprehensive list of our controls and their assemblies you can visit the [Installation and Deployment]({%slug installation-installing-which-file-do-i-need%}) section.
```
Comment (martinivanoff) [inline introduction.md]: You can use {{ site.framework_name }} here.

File: introduction.md [hunk 7]
```diff
@@ -41,32 +41,32 @@ License agreement and detailed information about different licenses and which on
...
+To learn how to install Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% if site.site_name == 'WPF' %}WPF{% endif %}{% endif %} and for comprehensive list of our controls and their assemblies you can visit the [Installation and Deployment]({%slug installation-installing-which-file-do-i-need%}) section.
 
 ## Consuming Data
 
-To learn more about the different approaches for populating the Telerik &reg; UI for {% if site.site_name == 'Silverlight' %} Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} controls, visit the [Consuming Data]({%slug consuming-data-overview%}) section.
+To learn more about the different approaches for populating the Telerik &reg; Telerik UI for {% if site.site_name == 'Silverlight' %} Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} controls, visit the [Consuming Data]({%slug consuming-data-overview%}) section.
```
Comment (martinivanoff) [inline introduction.md]: You can use {{ site.framework_name }} here.

File: introduction.md [hunk 8]
```diff
@@ -41,32 +41,32 @@ License agreement and detailed information about different licenses and which on
...
 
 ## Styling and Appearance
 
-To find how to customize the look of UI for {% if site.site_name == 'Silverlight' %} Silverlight{% if site.site_name == 'WPF' %}WPF{% endif %}{% endif %} using tools such as Expression Blend and information on how to use our predefined themes visit the [Styling and Appearance]({%slug styling-apperance-implicit-styles-overview%}) section.
+To find how to customize the look of Telerik UI for {% if site.site_name == 'Silverlight' %} Silverlight{% if site.site_name == 'WPF' %}WPF{% endif %}{% endif %} using tools such as Expression Blend and information on how to use our predefined themes visit the [Styling and Appearance]({%slug styling-apperance-implicit-styles-overview%}) section.
```
Comment (martinivanoff) [inline introduction.md]: You can use {{ site.framework_name }} here.

File: styling-and-appearance/styling-apperance-editing-control-templates.md [hunk 9]
```diff
@@ -10,14 +10,14 @@ position: 2
 
 # Editing Control Templates
 
-This article will show you two different approaches on how to extract and edit the default control templates of the UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} controls.
+This article will show you two different approaches on how to extract and edit the default control templates of the Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} controls.
```
Comment (martinivanoff) [inline styling-and-appearance/styling-apperance-editing-control-templates.md]: You can use {{ site.framework_name }} here.

File: styling-and-appearance/styling-apperance-editing-control-templates.md [hunk 10]
```diff
@@ -10,14 +10,14 @@ position: 2
...
 
 * [Extracting Control Templates Manually from the Theme XAML File](#extracting-control-templates-manually-from-the-theme-xaml-file)
 * [Extracting Control Templates Using Visual Studio](#extracting-control-templates-using-visual-studio)
 
 ## Extracting Control Templates Manually from the Theme XAML File
 
-Inside the installation folder of your version of UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} you can find a folder named __Themes.Implicit__. It contains the **XAML** files of the different themes for all the controls.
+Inside the installation folder of your version of Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} you can find a folder named __Themes.Implicit__. It contains the **XAML** files of the different themes for all the controls.
```
Comment (martinivanoff) [inline styling-and-appearance/styling-apperance-editing-control-templates.md]: You can use {{ site.framework_name }} here.

File: styling-and-appearance/styling-apperance-setting-a-theme-overview.md [hunk 11]
```diff
@@ -10,7 +10,7 @@ position: 0
 
 # Setting a Theme
 
-__UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ suite provides a [variety of themes]({%slug common-styling-appearance-available-themes%}) that will help you achieve outstanding visual appearance and great user experience. Before choosing what theme to apply and what approach to take in order to do so, you might find it useful to familiarize with the [Themes Concept](#what-is-a-theme) and their [Distribution](#where-can-a-theme-be-found).
+__Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ suite provides a [variety of themes]({%slug common-styling-appearance-available-themes%}) that will help you achieve outstanding visual appearance and great user experience. Before choosing what theme to apply and what approach to take in order to do so, you might find it useful to familiarize with the [Themes Concept](#what-is-a-theme) and their [Distribution](#where-can-a-theme-be-found).
```
Comment (martinivanoff) [inline styling-and-appearance/styling-apperance-setting-a-theme-overview.md]: You can use {{ site.framework_name }} here.

File: styling-and-appearance/styling-apperance-setting-a-theme-overview.md [hunk 12]
```diff
@@ -22,7 +22,7 @@ To apply a theme, you need to merge these files into the **Merged Dictionaries**
 
 ## Where Can a Theme Be Found?
 
-After installing the __UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ suite, you will find the following structure in the installation folder (usually C://Program Files/Telerik/ UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} [version]/)
+After installing the __Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ suite, you will find the following structure in the installation folder (usually C://Program Files/Telerik/ Telerik UI for {% if site.site_name == 'WPF' %}WPF{% endif %}{% if site.site_name == 'Silverlight' %}Silverlight{% endif %} [version]/)
```
Comment (martinivanoff) [inline styling-and-appearance/styling-apperance-setting-a-theme-overview.md]: You can use {{ site.framework_name }} here.

File: styling-and-appearance/styling-apperance-setting-a-theme-overview.md [hunk 13]
```diff
@@ -33,7 +33,7 @@ After installing the __UI for {% if site.site_name == 'Silverlight' %}Silverligh
 
 ### Themes.Implicit Folder
 
-As its name suggests, the **Themes.Implicit** folder contains all styles and resources needed for the visualization of the **UI for WPF** controls. Further digging into its structure, you will notice that it contains separate folders for each available theme.  
+As its name suggests, the **Themes.Implicit** folder contains all styles and resources needed for the visualization of the **Telerik UI for WPF** controls. Further digging into its structure, you will notice that it contains separate folders for each available theme.
```
Comment (martinivanoff) [inline styling-and-appearance/styling-apperance-setting-a-theme-overview.md]: for WPF? Isn't this also for Silverlight?

File: visual-studio-extensions/vs-extensions-options.md [hunk 14]
```diff
@@ -18,7 +18,7 @@ It can be accessed through the __Visual Studio | Telerik | VSExtensions Options
 
 ![VSExtentions Options Dialog Menu](images/VSExtentions_OptionsDialogMenu.png)
 
-The __Options__ dialog contains two sets of options that affect the __UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__.
+The __Options__ dialog contains two sets of options that affect the __Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__.
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-options.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-options.md [hunk 15]
```diff
@@ -35,7 +35,7 @@ The settings under the __General__ category affect all of the installed __Teleri
 
 __Suggest project upgrades for Telerik product version available on my computer__ – When enabled, you will be prompted to upgrade upon opening a project, which is not using the latest version of Telerik UI installed on your system
 
-__Suggest upgrades when an equal Dev release is detected on projects using a Trial__– When enabled, you will be prompted to upgrade if a licensed version of UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} is available on your system, but the current project uses a trial version
+__Suggest upgrades when an equal Dev release is detected on projects using a Trial__– When enabled, you will be prompted to upgrade if a licensed version of Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} is available on your system, but the current project uses a trial version
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-options.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-options.md [hunk 16]
```diff
@@ -44,7 +44,7 @@ Configures the path where the extensions look for and store distributions.
 
 >Changing the folder path will not move existing folder contents from your previous path. Please, move your previous folder contents manually in case you still want to use them.
 
-All settings under the UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} category affect only the respective product.
+All settings under the Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} category affect only the respective product.
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-options.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-options.md [hunk 17]
```diff
@@ -54,4 +54,4 @@ __Include internal builds in Latest Version update and retrieval__ – When enab
 
 ## Notifications
 
-__Show me a message when a newer version is available on [www.telerik.com](http://www.telerik.com)__ – When enabled, you will receive notifications if a new version of __UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ is available on the Telerik website.
+__Show me a message when a newer version is available on [www.telerik.com](http://www.telerik.com)__ – When enabled, you will receive notifications if a new version of __Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ is available on the Telerik website.
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-options.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-project-latest-version-acquirer.md [hunk 18]
```diff
@@ -14,10 +14,10 @@ position: 6
...
 
 
-{% if site.site_name == 'Silverlight' %}Once a day, upon Visual Studio launch, the Telerik Silverlight{% endif %}{% if site.site_name == 'WPF' %}When a solution containing an UI for WPF project is loaded in Visual Studio, the Telerik WPF{% endif %} VSExtensions queue the Telerik website for a new version of UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}. A dialog gets displayed when a new version is discovered:
+{% if site.site_name == 'Silverlight' %}Once a day, upon Visual Studio launch, the Telerik Silverlight{% endif %}{% if site.site_name == 'WPF' %}When a solution containing an Telerik UI for WPF project is loaded in Visual Studio, the Telerik WPF{% endif %} VSExtensions queue the Telerik website for a new version of Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}. A dialog gets displayed when a new version is discovered:
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-project-latest-version-acquirer.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-scenario-templates.md [hunk 19]
```diff
@@ -14,7 +14,7 @@ position: 5
 
 ## 
 
-The Scenario templates’ purpose is to let you quickly start using UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} in your existing projects. You can use the following steps to use our Scenario templates.
+The Scenario templates’ purpose is to let you quickly start using Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} in your existing projects. You can use the following steps to use our Scenario templates.
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-scenario-templates.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-scenario-templates.md [hunk 20]
```diff
@@ -36,7 +36,7 @@ The Scenario templates’ purpose is to let you quickly start using UI for {% if
 
 	>Currently there are three predefined scenarios: 
 	
-	* __Page__ – Creates a basic Page, ready to use with UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}
+	* __Page__ – Creates a basic Page, ready to use with Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-scenario-templates.md]: You can use {{ site.framework_name }} here.

File: visual-studio-extensions/vs-extensions-toolbox-configurator.md [hunk 21]
```diff
@@ -14,7 +14,7 @@ position: 7
 
 ## 
 
-Starting with __Q3 2010__ we introduced the __Toolbox Configuration__ utility. Its purpose is to choose which version of __UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ to be populated in the Visual Studio’s Toolbox. The user can select among all versions of __Telerik UI__ that are currently on the machine and have been either installed or downloaded using the __VS Extensions__.
+Starting with __Q3 2010__ we introduced the __Toolbox Configuration__ utility. Its purpose is to choose which version of __Telerik UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %}__ to be populated in the Visual Studio’s Toolbox. The user can select among all versions of __Telerik UI__ that are currently on the machine and have been either installed or downloaded using the __VS Extensions__.
```
Comment (martinivanoff) [inline visual-studio-extensions/vs-extensions-toolbox-configurator.md]: You can use {{ site.framework_name }} here.

File: eqatec-analytics-application-integration.md (hunk 3 above)
Comment (dilyantraykov) [inline eqatec-analytics-application-integration.md]: This resides in an "if Silverlight" condition.

**Pull request merged**